apps/token-e2e/src/integration/flow/token-association-flow.cy.js

```diff
@@ -54,6 +70,11 @@ context(
       });
 
       it('Able to disassociate tokens', function () {
+        // 1000-ASSO-0025
```

Comment:
question (non-blocking): i feel dumb for asking, but what are these comments here?

Reply:
AC coverage :)

Reply:
https://github.com/vegaprotocol/approbation

These allow us "functionality" test coverage, instead of lines of code coverage.

Reply:
makes sense